.github/workflows/molecule.yml

```diff
@@ -21,7 +42,7 @@ jobs:
       - name: Check out code
         uses: actions/checkout@v3
 
-      - name: Set up Python 3.9
+      - name: Set up Python 3.8
```

Comment:
Is there a reason why we need an old python version?

Reply:
Sorry, just read your comment in https://github.com/NETWAYS/ansible-role-redis-cluster/pull/15#issuecomment-1597742392